tests/run/kwargs_passthrough.pyx

```diff
@@ -1,22 +1,13 @@
 cimport cython
 
 
-@cython.test_fail_if_path_exists('//MergedDictNode')
-def wrap_passthrough(f):
-    """
-    >>> def f(a=1): return a
-    >>> wrapped = wrap_passthrough(f)
-    >>> wrapped(1)
-    CALLED
-    1
-    >>> wrapped(a=2)
-    CALLED
-    2
-    """
-    def wrapper(*args, **kwargs):
-        print("CALLED")
-        return f(*args, **kwargs)
-    return wrapper
+def modify_in_function():
+    def inner(**kwds):
```

Comment:
```suggestion
def modify_in_function():
    """
    >>> modify_in_function()
    {'foo': 'bar'}
    {'foo': 'bar'}
    """
    def inner(**kwds):
```

My fault for not being clear when I suggested it - this does need a doctest so that it gets run.